Modules/CalcOffence.lua

```diff
@@ -1735,6 +1735,16 @@ function calcs.offence(env, actor, activeSkill)
 		output.DoubleDamageEffect = 1 + output.DoubleDamageChance / 100
 		output.ScaledDamageEffect = output.ScaledDamageEffect * output.DoubleDamageEffect
 		
+		-- Calculate culling DPS
+		local criticalCull = skillModList:Max(cfg, "CriticalCullPercent") or 0
+		if criticalCull > 0 then
+			criticalCull = criticalCull * (output.CritChance / 100)
+		end
+		local regularCull = skillModList:Max(cfg, "CullPercent") or 0;
+		local maxCullPercent = m_max(criticalCull, regularCull);
+		globalOutput.CullPercent = maxCullPercent;
+		globalOutput.CullMultiplier = 100 / (100 - globalOutput.CullPercent)
+
```

Comment:
```suggestion
		globalOutput.CullMultiplier = 100 / (100 - globalOutput.CullPercent)
```
Style: avoid excessive empty lines.